lib/models/actions.js

```diff
@@ -62,6 +63,9 @@ function conditionalExec(action, event, lastTime, callbackW) {
                 case 'post':
                     postAction.doIt(action, event, callbackS);
                     break;
+                case 'twitter':
```

Comment:
I think this code could be improved creating an object with the action name as an attribute and the module as the value, so all this switch could be summarized into:  actionMap[action.type].doIt(action, event, callbackS) (with an if for the case where actionMap[action.type]). This would make introducing a new action easier, and it could even be modified to have it programmatically.